pkgs/by-name/ch/chrome-remote-desktop/package.nix

```
@@ -0,0 +1,114 @@
+{ stdenvNoCC
+, lib
+, autoPatchelfHook
+, dpkg
+, fetchurl
+, glib
+, gtk3
+, libdrm
+, libutempter
+, mesa
+, nss
+, pam
+, python3
+, shadow
+, xorg
+}:
+
+let
+  replacePrefix = "/opt/google/chrome-remote-desktop";
+in
+stdenvNoCC.mkDerivation rec {
+  name = "chrome-remote-desktop";
+  # Get the latest version from:
+  # https://dl.google.com/linux/chrome-remote-desktop/deb/dists/stable/main/binary-amd64/Packages
+  version = "120.0.6099.24";
+  src = fetchurl {
+    url = "https://dl.google.com/linux/chrome-remote-desktop/deb/pool/main/c/chrome-remote-desktop/chrome-remote-desktop_${version}_amd64.deb";
+    hash = "sha256-HA1sMR/v3b7u1GaCTzqdAV4Ov5mrRoYU4vPdyEYnX2Y=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    glib
+    gtk3
+    libdrm
+    libutempter
+    mesa
+    nss
+    pam
+    xorg.libX11
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXrandr
+    xorg.libXtst
+  ];
+
+  dontBuild = true;
+  dontConfigure = true;
+
+  unpackPhase = ''
+    runHook preUnpack
+
+    ${dpkg}/bin/dpkg -x $src $out
+
+    runHook postUnpack
+  '';
+
+  patchPhase = ''
+    runHook prePatch
+
+    sed \
+      -e '/^.*sudo_command =/ s/"gksudo .*"/"pkexec"/' \
+      -e '/^.*command =/ s/s -- sh -c/s sh -c/' \
+      -i $out/opt/google/chrome-remote-desktop/chrome-remote-desktop
+
+    substituteInPlace $out/lib/systemd/system/chrome-remote-desktop@.service \
+      --replace /opt/google/chrome-remote-desktop/chrome-remote-desktop '${placeholder "out"}/bin/chrome-remote-desktop'
+
+    substituteInPlace $out/etc/opt/chrome/native-messaging-hosts/com.google.chrome.remote_desktop.json \
+      --replace ${replacePrefix}/native-messaging-host $out/${replacePrefix}/native-messaging-host \
+      --replace ${replacePrefix}/remote-assistance-host $out/${replacePrefix}/remote-assistance-host
+
+    substituteInPlace $out/${replacePrefix}/chrome-remote-desktop \
+      --replace /usr/bin/python3 ${python3.withPackages (ps: with ps; [ psutil pyxdg packaging ])}/bin/python3 \
```

it could be in the `let-in` above, like
```suggestion
      --replace /usr/bin/python3 ${lib.getBin pythonEnv}/bin/python3 \
```